Vic2ToHoI4/Data_Files/blankMod/output/portraits/00_portraits.txt

```diff
@@ -9874,27 +9951,6 @@ continent = {
 			"gfx/leaders/Arabia/portrait_arabia_generic_operative_male_4.dds"
 			"gfx/leaders/Arabia/portrait_arabia_generic_operative_male_5.dds"
 			"gfx/leaders/Arabia/portrait_arabia_generic_operative_male_6.dds"
-			"gfx/leaders/JOR/Portrait_Arabia_Generic_1.dds"
```

Comment:
Just to double-check, removing these (and all the ones below) was intentional, right?